src/components/Navbar/Navbar.css

```diff
@@ -1,31 +1,57 @@
+
 .navBar {
+  width: 100vh;
+  width: 100%;
+}
```

Comment:
- [ ] You are using the same property twice here which would make confusion, please use the correct one.

Reply:
Noted